index.html

```diff
@@ -81,6 +81,235 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
 
         </div>
 
+
+<!-- project -->
+         <div id="portfolio" class="cards grid-container">
+            <section class="first-card card">
+                <img class="snapshoot-style" src="images/Snapshoot Portfolio.svg" alt="Snapshoot portfolio image">
+                <section>
+                    <h2 class="card-header font-style on-small">
+                        Tonic
+                    </h2>
+                    <h2 class="on-large card-header font-style">
+                        Facebook 360
+                    </h2>
+                    <section class="card-description">
+                        <ul class="font-style">
+                            <li class="card-list-title on-small"> CANOPY</li>
+                            <li class="on-large card-list-title ">FACEBOOK</li>
+                            <li class="on-large Job-title_and_year">Full Stack Dev</li>
+                            <li class="Job-title_and_year on-small">
+                                Back End Dev
+                            </li>
+                            <li class="Job-title_and_year">
```

Comment:
- [ ] Kindly consider the container of the works section is a section tag and for each project using a div tag.